ipi_tests/examples/exampletools.py

```diff
@@ -59,6 +67,9 @@ def get_driver_info(
         "socket_mode": socket_mode,
         "port_number": port_number,
         "address_name": address_name,
+        "LJ_flags1": flags1,
```

Comment:
It would be better to make this more general, i.e. not only for LJ potential

Reply:
see general comment below